developers/blobstream-proof-queries.md

```diff
@@ -999,9 +1000,7 @@ func submitFraudProof(
 	dataRoot []byte,
 ) error {
 	var blockDataRoot [32]byte
-	for i, b := range dataRoot[58:] {
-		blockDataRoot[i] = b
-	}
+	copy(blockDataRoot[:], dataRoot)
```

Comment:
We can use the data root directly without making a copy. For some reason, I thought that the data root would also contain the min-max namespace. Good catch 🙏